arkouda/numeric.py

```diff
@@ -508,3 +508,28 @@ def value_counts(pda : pdarray) -> Union[Categorical, # type: ignore
     (array([0, 2, 4]), array([3, 2, 1]))
     """
     return unique(pda, return_counts=True)
+
+
+@typechecked
+def isna(pda : pdarray) -> pdarray:
```

Comment:
@glitch the more I think about this the more I think that the first thing is to use the numpy interface for `np.isnan()` for `float64`, then later we can build up the pandas interface on top of these functions.

Reply:
@mhmerrill RE: alter `int64` and `strings` yes, I was thinking along similar lines.  We'd almost have to encapsulate int64 in our own object which would be something like `class Int64(value:int64, missing:bool)`.  I haven't looked at their implementation but I think this is roughly what Pandas ended up doing.

Regarding float64 do you want me to change the `isna` to `isnan`?

Reply:
yes change `isna` to `isnan`